pages/common/mysqld.md

```diff
@@ -0,0 +1,32 @@
+# mysqld
+
+> Start the MariaDB database server.
```

Comment:
It's funny to have this refer to `MariaDB` in help text, but link to mysql docs. Probably better to just say `MySQL` and people using `MariaDB` should know that they're largely interchangeable.